security/README.md

```diff
@@ -0,0 +1,19 @@
+
+
+# Network Security Policy
+
+Sample policy to allow two namesapces to talk; future version would be able to apply a wildcard like `devhub-*`. The operator will ensure the originating matches the source. The destination namespace will need to add a similar rule with the source and destination reversed to enable bi-directional communication.
+
+```yaml
+apiVersion: secops.pathfinder.gov.bc.ca/v1alpha1
+kind: SecOpsPolicy
```

Comment:
@stewartshea I've assumed we'll just have one `SecOpsPolicy` CRD. We could use `NetSecOps` to be more clear that it is a networking policy and should be fed into Aporeto. It'll still be dealt with by the same operator; I'm not sure how complicated things can get with Ansible. If they get too weird it could always be re-created in Go for more flexibility.

Reply:
I prefer something closer to `NetworkSecurityPolicy` for the kind here to make it more specific. I anticipate having multiple types here as we evolve the platforms capability and this will make it network specific without locking us into the vendor.

Reply:
Renamed.